core/deepmind.py

```
@@ -427,6 +427,34 @@ async def organize_and_inject_memories(
                 self.user_profile_service.set_memory_sql_manager(memory_sql_manager)
+                # 登记当前事件用户与群聊到账本
+                ledger_sender_id, ledger_sender_name = self._get_event_sender_identity(event)
+                if ledger_sender_id and ledger_sender_id.lower() not in {
+                    "unknown", "user", "assistant", "none", "null", "0",
+                }:
+                    try:
+                        ledger_platform = ""
+                        if hasattr(event, "get_platform_name"):
+                            try:
+                                ledger_platform = str(event.get_platform_name() or "").strip()
+                            except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)